change_notes/2024-03-01-fix-fp-a12-4-1-and-a12-8-6.md

```diff
@@ -0,0 +1,6 @@
+- `A12-4-1` - `DestructorOfABaseClassNotPublicVirtual.ql`:
+  - Fix FP reported in #392. Improve base class detection for template classes.
+  - Update the alert message to prevent duplicate alerts for base classes that are both derived and abstract.
+- `A12-8-6` - `CopyAndMoveNotDeclaredProtected.ql`:
+  - Fix FP reported in #392. Improve base class detection for template classes.
```

Comment:
super small nitpick, but it was a FN (not FP?) here, I am pretty sure?

Reply:
We incorrectly detected template base classes resulting in false positives on subclasses that didn't adhere to the rules for copy/move constructors/operators.